federatedscope/core/fed_runner.py

```diff
@@ -160,7 +164,7 @@ def _setup_server(self, resource_info=None, client_resource_info=None):
                 wrap_nbafl_server
             wrap_nbafl_server(server)
         logger.info('Server has been set up ... ')
-        return server
+        return self.feat_engr_wrapper_server(server)
```

Comment:
It is ok to inject those feature engineering procedures into an FL course in this way, but we shall change the instantiation of workers to a better (more general and unified) way. Actually, it is quite confusing to wrap a worker by feature engineering-related wrapper. Feature engineering is just a tiny step in an FL course, which doesn't change a worker significantly. One usual way to instantiate a worker from a collection of such pluggable behaviors is to use factory pattern I guess.